rocketchat-common/src/main/java/com/rocketchat/common/data/lightstream/document/RocketChatRolesDocument.java

```diff
@@ -0,0 +1,88 @@
+package com.rocketchat.common.data.lightstream.document;
+
+import java.util.Date;
+import org.json.JSONObject;
+
+public class RocketChatRolesDocument {
+    String name;
+    String scope;
+    String description;
+    Boolean _protected;
+    Date updatedAt;
```

Comment:
Lets use `Long` for dates (timestamps) and let the user decide if it wants to use `Date` or the new JSR-310 types or JodaTime